src/ui/components/pages/assets/assetInfo.tsx

```diff
@@ -0,0 +1,149 @@
+import { Trans } from 'react-i18next';
+import { Balance, Button, Copy, DateFormat, Ellipsis } from '../../ui';
+import * as React from 'react';
+import { Asset, Money } from '@waves/data-entities';
+import { useAppSelector } from '../../../store';
+
+interface AssetDetail extends Asset {
+  originTransactionId: string;
+}
+
+interface Props {
+  asset: AssetDetail;
+  onCopy: () => void;
+  onClose: () => void;
+}
+
+export function AssetInfo({ asset, onCopy, onClose }: Props) {
+  const selectedAccount = useAppSelector(state => state.selectedAccount);
+
+  const network = selectedAccount && selectedAccount.networkCode;
```

Comment:
I doubt `selectedAccount` can be falsy here. Also, you can get `networkCode` right in the selector. And I would name it `networkCode` as well. I guess it's a good idea to keep `network` for something like "mainnet" or "testnet" and `networkCode` for "W" or "T". So, something like this should work:

```ts
const networkCode = useAppSelector(state => state.selectedAccount.networkCode);
```